gradle/spotbugs/exclude.xml

```diff
@@ -279,6 +279,7 @@
       <Class name="com.linkedin.davinci.store.blackhole.BlackHoleStorageEnginePartition"/>
       <Class name="com.linkedin.davinci.store.memory.InMemoryStoragePartition"/>
       <Class name="com.linkedin.davinci.store.rocksdb.RocksDBStoragePartition"/>
+      <Class name="com.linkedin.venice.client.schema.RouterBackedSchemaReader"/>
```

Comment:
I don't see much benefit in returning empty `byte[]` vs `null` in our case since we immediately pass the `byte[]` to deserialize as JSON. If the object that we're deserializing to has mandatory fields, then there will be an exception thrown due to deserialization error.

To avoid this, we would be checking `return.length == 0` instead of `return == null`. We would still need to keep our null checks in the callers of the function (or return sentinel values).

Reply:
Sure, but this change will exclude the spotbugs checks for the future changes as well, right?
Seems to be a bad thing.
Can you share the exact function you would like to skip the spotbug checks? 
If you have a strong desire to skip it, maybe put it in a separate class and only skip that class, but don't exclude this big class: `RouterBackedSchemaReader`?

Reply:
Oh, it only excludes it for `PZLA_PREFER_ZERO_LENGTH_ARRAYS` check. The `executeRouterRequest` function is getting caught